packages/slice-machine/server/src/api/services/sliceService.ts

```diff
@@ -51,7 +49,7 @@ export async function uploadScreenshots(
 
     if (!!screenshot) {
       const { err, s3ImageUrl }: { err?: ApiError; s3ImageUrl?: string } =
-        await upload(env, sliceName, variationId, screenshot.path);
+        await upload(env, sliceModel, variationId, screenshot.path);
```

Comment:
why are you passing the whole model down? Think it's better if you just pass down the slice ID if that's all you need

Reply:
That would make the tests easier as well.